Include/cpython/genobject.h

```diff
@@ -26,7 +28,7 @@ extern "C" {
     char prefix##_running_async;                                            \
     /* The frame */                                                         \
     int8_t prefix##_frame_state;                                            \
-    PyObject *prefix##_iframe[1];                                           \
+    struct _PyInterpreterFrame *prefix##_iframe[1];                         \
```

Comment:
I tried to remove the [1] but then it crashes during the build. Not sure why.

Reply:
```suggestion
    struct _PyInterpreterFrame prefix##_iframe;                         \
```
It's not a pointer to a frame, it is a frame.

Reply:
> Given that, maybe the object size calculation could be simplified to not need the explicit addition (not tested or 100% sure on that, though): https://github.com/python/cpython/blob/main/Objects/genobject.c#L818

That seems to work.